crates/bevy_core/src/time/time.rs

```diff
@@ -31,7 +31,8 @@ impl Time {
         self.update_with_instant(now);
     }
 
-    pub(crate) fn update_with_instant(&mut self, instant: Instant) {
+    /// Update time with a specified [`Instant`], for use in tests. Should never be called on the default Time resource.
```

Comment:
```suggestion
    /// Update time with a specified [`Instant`]
    ///
    /// This method is for use in tests, and should never be called on the default Time resource.
```

Reply:
Just inaccurate timekeeping in general. I would keep the recommendation to *only* use this to write tests.

Reply:
Just inaccurate timekeeping in general. I would keep the recommendation to *only* use this to write tests.